packages/components/src/navigation/style.scss

```diff
@@ -0,0 +1,34 @@
+.components-navigation {
+	width: 272px;
+	padding: 35px 30px;
+	background: #191e23;
+	color: #ccd0d4;
+}
+
+.components-navigation-back {
+	margin-bottom: 64px;
+	background-color: #23282d;
+	padding: 3px 6px;
+	color: #ccd0d4;
+}
+
+.components-navigation-title {
+	// For now
+	margin-bottom: 40px;
+}
+
+.components-navigation-items {
+	display: flex;
+	flex-direction: column;
+}
+
+.components-navigation-item {
```

Comment:
Menu items with long text may be capped by height and I think we should left-align the text here.  Default height in designs looks to be about 40px.

<img width="247" alt="Screen Shot 2020-07-22 at 1 13 25 PM" src="https://user-images.githubusercontent.com/10561050/88164861-68a54180-cc1d-11ea-8948-0aed248997de.png">

Reply:
Do we also want to remove the focus outline on menu items here?

<img width="263" alt="Screen Shot 2020-07-22 at 1 10 42 PM" src="https://user-images.githubusercontent.com/10561050/88165029-bb7ef900-cc1d-11ea-986f-a1792bd375b2.png">

Reply:
I think we'll need to keep focus outline for keyboard users. At least change the color or something to make it more presentable.

Good point on the long names, that will certainly be an issue.